example/spec/parallel/testAPI/ruleSpec.js

```diff
@@ -248,7 +260,16 @@ describe('When I create a one-time rule via the Cumulus API', () => {
         findExecutionFnParams: { rule: createdCheck },
         startTask: 'HelloWorld',
       });
-      console.log(`Execution ARN: ${execution.executionArn}`);
+
+      executionArn = execution.executionArn;
+      console.log(`Execution ARN: ${executionArn}`);
+    });
+
+    afterAll(async () => {
+      await Promise.allSettled([
+        deleteExecution({ prefix: config.stackName, executionArn: executionArn }),
```

Comment:
Oops - commented on the commit, re-posting: 

We should probably still be checking for a  rejected promise in the array here and logging/throwing/etc, or it'll be *really* hard to tell what's going on if it fails.

Reply:
👍🏻  to switch back to `.all`, in lieu of fixing this more broadly in the integration tests.